backend/amd64/cfg_selection.ml

```diff
@@ -353,7 +353,6 @@ let select_operation
     match func with
     | "caml_rdtsc_unboxed" -> Rewritten (specific Irdtsc, args)
     | "caml_rdpmc_unboxed" -> Rewritten (specific Irdpmc, args)
-    | "caml_pause_hint" -> Rewritten (specific Ipause, args)
```

Comment:
I think we should keep this intrinsic since `%cpu_relax` also acts as a polling point when poll insertion is disabled. It can be recognized in `cmm_builtins` and mapped to `Crelax` now.

Reply:
ah, I missed that part, thanks! fixed.